aerospike-storage-backend/src/main/java/com/playtika/janusgraph/aerospike/AerospikeStoreManager.java

```diff
@@ -80,10 +102,12 @@ public AerospikeStoreManager(Configuration configuration) {
         writeAheadLogCompleter.start();
 
         scanExecutor = new ThreadPoolExecutor(0, configuration.get(SCAN_PARALLELISM),
```

Comment:
maybe it is better to name thread pools differently? to be able to differentiate scanExecutor from aerospikeExecutor